src/lib/blueprint.ts

```diff
@@ -64,6 +65,11 @@ export interface Resource {
   undocumentedMessage: string
   isDraft: boolean
   draftMessage: string
+  propertyGroups: Record<string, PropertyGroup>
+}
+
+interface PropertyGroup {
+  name: string
```

Comment:
Putting the key here will help consumers like console that just want to loop over `Object.keys`
```suggestion
  name: string
  propertyGroupKey: string
```

Reply:
c4de502aa3d35a89a623ab03df4bd728938c05f0